pages/content/amp-dev/documentation/examples/index.html

```diff
@@ -30,30 +30,194 @@
     <span id="ap--sidebar-content" class="ap--sidebar"></span>
     <section class="ap--content">
       {% include '/views/partials/sidebar-toggle-button.j2' %}
+      {% include '/views/partials/breadcrumbs.j2' %}
+
       <h1>Learn AMP by example</h1>
+      <p>A hands-on introduction to AMP focusing on code and live samples. Learn how to create AMP pages and see examples for all AMP components.</p>
+
+      {% do doc.styles.addCssFile('/css/components/organisms/card-grid.css') %}
+      <section class="ap--card-grid">
+        <h2 class="ap-o-card-grid-text">AMP’s examples</h2>
+
 
-      <amp-state id="activeFilter">
-        <script type="application/json">
+        {% with %}
+        {% set card_grid_items = [
           {
-            "chosenFilter": "none"
+            "icon_name": "code",
+            "headline": "Components",
+            "text": "Create every content, layout and feature on your website with a amp component.",
+            "strong": "These are the most used components:",
```

Comment:
Where is this coming from? I fear we have no data for that, also I'd wonder if the listed ones are really the most used ones 😄

Reply:
Text and chosen examples came from the design. Should we change the phrase an/or examples? 😅

Reply:
Pinged Sebastian in another comment, let's see what he comes up with 🙂